src/client/search/AddSearchTerm.js

```diff
@@ -22,12 +22,15 @@ let AddSearchTerm = ({ dispatch }) => {
         onBlur={() => {
           $('#searchTermContainer').slideUp('fast');
         }}
+
         onKeyDown={(e) => {
           if (e.keyCode === 13) {
             dispatch(addSearchTerm(nextSearchTermId++, e.target.value));
+            dispatch(invalidateFeedResults());
```

Comment:
Like you said for a minimum right now this is fine and will refactor another time... Opening issue for it now